Convert grep package to DOWNLOAD helper.

Not needed on recent BR.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: buildroot/package/grep/grep.mk
===================================================================
--- buildroot.orig/package/grep/grep.mk	2010-11-24 11:50:09.000000000 +0100
+++ buildroot/package/grep/grep.mk	2010-11-24 11:51:20.000000000 +0100
@@ -12,7 +12,7 @@
 GNUGREP_TARGET_BINARY:=bin/grep
 
 $(DL_DIR)/$(GNUGREP_SOURCE):
-	 $(WGET) -P $(DL_DIR) $(GNUGREP_SITE)/$(GNUGREP_SOURCE)
+	$(call DOWNLOAD,$(GNUGREP_SITE),$(GNUGREP_SOURCE))
 
 grep-source: $(DL_DIR)/$(GNUGREP_SOURCE)
 
